### ISSUE TITLE
Reolink Wifi Video Doorbell  iOS homekit notification

### ISSUE THREAD

**Issue by lishan89uc** [koush/scrypted#1511]
**Describe the bug**
Reolink Wifi Video Doorbell no longer sending doorbell press notification  to Apple Homekit/Home App

**To Reproduce**
1. update [Reolink Camera Plugin] from 0.66 to 0.71
2. reload plugin if needed
3. press doorbell 
4. No doorbell press notification (note: motion notification still works

**Expected behavior**
Get a doorbell notification from home app stating someone ring the doorbell

**Screenshots**
N/A

**Server (please complete the following information):**
 - Synology
 - Docker(Container Manager)

**Hardware Model (please complete the following information):**
 - Reolink Video Doorbell WiFi

**Client (please complete the following information, if applicable):**
 - Home app

**Additional context**
I noticed the "This camera is a Reolink Doorbell." check box is gone, not sure if it is related

**Comment by koush** [comment]
fixed

**Comment by sabaatworld** [comment]
I'm using 0.74 and this does not seem to be fixed yet

**Comment by EthanBezz** [comment]
Same issue here, but was able to fix it.

I tried downgrading both the Reolink and HomeKit plugins to 0.0.66 and 1.2.57, respectively, but that made no difference. I then noticed that Scrypted wasn't even seeing any events (motion, binary, ect.) from the doorbell, so I restarted the doorbell and that fixed the issue. Finally, I updated both plugins back to their latest versions and re-tested and the issue was still fixed.

TL;DR
Reboot the doorbell. Maybe keep "Auto Reboot" enabled, too.

**Comment by budisek** [comment]
Hi

I have same problem. I’ve restarted Reolink doorbell, plugin, Scrypted addon and even home assistant but no motion and ringing happened. 

v0.0.94 version of Reolink plugin.

**Comment by budisek** [comment]
Any solution?

**Comment by lishan89uc** [comment]
> Any solution?

This issue is supposed to be fixed. I have v0.0.94 and it is working. Please check if you have the this is a doorbell box checked.

**Comment by ishanhanda** [comment]
> This issue is supposed to be fixed. I have v0.0.94 and it is working. Please check if you have the this is a doorbell box checked.

Running into this on v0.118.0. Tried everything suggested on other threads here. I get a single motion notification after restarting the ring plugin and nothing after that.
